README.md

```diff
@@ -1,16 +1,225 @@
 # spa-orders
 
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Scripts](#scripts)
+- [Git flow](#git-flow)
+- [Project structure](#project-structure)
+- [Code conventions](#code-conventions)
+- [Styling](#styling)
+- [References](#references)
+
 ## Getting Started
 
-1. `nvm use`
-2. `npm install`
-3. `npm run start`
+1. Use node version from `.nvmrc`
+
+```
+nvm use
+```
+
+2. Install dependencies
+
+```
+npm install
+```
+
+3. Start the project
+
+```
+npm run start
+```
 
-Open [http://localhost:3030](http://localhost:3000) with your browser to see the result.
+4. Open [http://localhost:3030](http://localhost:3000) with your browser to see the result.
 
 ## Scripts
 
-- `npm run lint` to lint the project files.
-- `npm run test` to run unit tests.
-- `npm run test:mutation` to run mutation tests.
-- `npm run typecheck` to check the validity of types.
\ No newline at end of file
+| Name                       | Description                              |
+| -------------------------- | ---------------------------------------- |
+| `npm run test`             | Runs unit tests                          |
+| `npm run test:coverage`    | Runs unit tests with collecting coverage |
+| `npm run test:mutation`    | Runs mutation tests                      |
+| `npm run typescript:check` | Checks the validity of types             |
+| `npm run lint`             | Lints the project files                  |
+| `npm run lint:fix`         | Fixes lint errors in the project         |
+
+## Git flow
+
+### Pull requests
+
+- Use `git rebase` to merge brances locally.
+- Use `Squash and merge` option to merge PR into develop
+
+### Main branches
+
+- `main` - production branch
+- `develop` - development branch
+
+### Branch naming
+
+Use pattern: `<prefix>/<issue_id>/<short_description>`
+
+For example: `feature/41/navbar`
+
+Allowed prefixes are:
+
+- `feature` - for new feature
+- `hotfix` - for urgent fixes
+- `bugfix` - for fixing existing functionality
```

Comment:
'tests' also can be I think

Reply:
Added test branch prefix. We can use it for unit, mutation and e2e tests